inline_scan.sh

```diff
@@ -523,45 +527,30 @@ get_scan_result_pdf_by_digest() {
 
 save_and_copy_images() {
     # Save all image files to /tmp and copy to created container
-    for image in "${SCAN_IMAGES[@]-}"; do
-        local base_image_name="${image##*/}"
-        echo "Saving ${image} for local analysis"
-        local save_file_name="${base_image_name}.tar"
-        if [[ ! "${image}" =~ [:]+ ]]; then
```

Comment:
since we are now using FULLTAG, all these checks are already done and don't need to be reinforced